### PR TITLE
Use empty prefix for gRPC package names

### PR DESCRIPTION
**Pull request by daviddrysdale**
# Checklist

- [x] Pull request affects core Oak functionality (e.g. runtime, SDK, ABI)
  - [x] I have written tests that cover the code changes.
  - [ ] I have checked that these tests are run by
        [Cloudbuild](/cloudbuild.yaml)
  - [ ] I have updated [documentation](/docs/) accordingly.
  - [ ] I have raised an [issue](https://github.com/project-oak/oak/issues) to
        cover any TODOs and/or unfinished work.
- [ ] Pull request includes prototype/experimental work that is under
      construction.

### CODE REVIEW

**Comment by daviddrysdale** [comment]
@ipetr0v do you remember why `""` didn't work previously?

File: examples/translator/proto/translator.proto [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 syntax = "proto3";
 
-package oak.examples.translator;
```
Comment (ipetr0v) [inline examples/translator/proto/translator.proto:19]: Should other examples remove `oak.` too?

**Comment by ipetr0v** [comment]
> @ipetr0v do you remember why "" didn't work previously?
I added this after the following comment:
https://github.com/project-oak/oak/issues/751#issuecomment-603829745

File: examples/translator/proto/translator.proto (hunk 1 above)
Comment (daviddrysdale) [reply examples/translator/proto/translator.proto:19]: I don't think it's needed, as long as there's one example that checks that "oak" isn't required.  (Added a comment).

**Pull request merged**